src/components/AssetsTableContent.jsx

```diff
@@ -8,6 +8,13 @@ import LoaderComponent from './LoaderComponent';
 import { ToastMessage } from '../_utils/ToastMessage';
 import NotFound from './common/ItemsNotFoundComponent';
 
+const getAssignee = (data) => {
+  if (!data) {
+    return '-';
+  }
+  return data.email || data.full_name || data.name || '-';
+};
+
```

Comment:
I think there may be a more concise way to express this. Please let me know if you have a better idea 👍🏽

Reply:
I can't think of a better way than the logical OR right now. It doesn't look that bad so let it be for now